### DISCUSSION TITLE
Asking for better Documentation on Offset Settings in Custom Mode

### DISCUSSION THREAD

**Discussion by Fenryl92** [BartoszCichecki/LenovoLegionToolkit#1014] [Q&A] [answered]
(First time on Github - Hope this belongs here?)

First of all the Lenovo Legion Toolkit is an amazing Tool - Thanks very much!

I could not find any Information on the Max- and Min- Offset function in Custom Mode within the existing documentation. Could you provide us with more insight on how this affects the Power Mode Settings? Does it over-/ undervolt the CPU/GPU? Any help would be greatly appreciated.

**Comment by BartoszCichecki** [comment, marked as answer]
Max/min offset is quite simple. Let's say you want to modify any setting in available in custom mode: as an example let's focus on CPU Short Term PL. 

The default range of values is between 25W and 90W. This is the range you can use as per spec that Lenovo provides. If you set minimum Offset to 10W and maximum offset to 20W, you will be able to set values from 15W to 110W.

It does not mean this value will actually be set and respected by the EC, but you can try. It is very inconsistent how "out-of-spec" values work, but it give some extra flexibility in fine tunning custom mode parameters.
